troubleshooting.md

```diff
@@ -2,6 +2,12 @@
 
 In addition to the [documentation](http://docs.heptio.com/content/aws.html) available for the Heptio AWS Quick Start, this page outlines some troubleshooting tips if you're running into issues with your quick start.
 
+## Quick Start CloudFormation Master Does Not Start Properly
```

Comment:
What do you think about mentioning the cloudwatch logs as well?

Reply:
Thanks @chuckha didn't know we actually exported that log into the CloudWatch Logs group.